### ISSUE TITLE
Using a wide angle lens

### ISSUE THREAD

**Issue by YapCK45** [s60sc/ESP32-CAM_MJPEG2SD#139]
I recently bought a wide angle lens (160 deg) and replaced the standard lens (70 deg) . Cannot get the motion detection to work. Seems like when everything becomes smaller, the parameters for motion detection need to be tweaked. Would appreciate it if there is some explanation of these parameters.

**Comment by s60sc** [comment]
use _Show Motion_ feature to monitor, increase _Motion sensitivity_, reduce _Pixel difference to indicate change_ under _Motion_ in _Edit Config_ tag

**Comment by YapCK45** [comment]
I have tried that. _Pixel difference_ is quite intuitive. I have reduced _Num changed frames to start motion_ and increased _Checks per second for start motion_.  I reckoned that cannot be more than the fps setting in Camera Control.
What are _Bottom band where motion is checked_ and _Top band where motion is checked_?  Are they related to _Total number of detection bands_?
This is my configuration:
![image](https://user-images.githubusercontent.com/64215797/197942127-6dc2f77c-f369-4af4-8495-654306397b9c.png)
